query-node/schemas/council.graphql

```diff
@@ -0,0 +1,281 @@
+  electionRound: ElectionRound!
+
+  "Stake locked for the candidacy."
+  stake: BigInt!
+
+  "Reflects if the stake is still locked for candidacy or has been already released by the member."
+  stakeLocked: Boolean!
+
+  "Reflects if the candidacy was withdrawn before voting started."
+  candidacyWithdrawn: Boolean!
+
+  "Sum of power of all votes received."
+  votePower: BigInt!
+
+  "The metadata contained in note."
+  noteMetadata: CandidacyNoteMetadata!
+}
+
+type CouncilMember @entity {
+  "Runtime council member id"
+  id: ID!
+
+  "Account used for staking currency for council membership."
+  stakingAccountId: String!
+
+  "Account that will receive used for reward currency for council membership."
+  rewardAccountId: String!
+
+  "Council member's membership."
+  member: Membership!
+
+  "Stake used for the council membership."
+  stake: BigInt!
+
+  "Block number in which council member recieved the last reward payment."
+  lastPaymentBlock: BigInt!
+
+  "Reward amount that should have been paid but couldn't be paid off due to insufficient budget."
+  unpaidReward: BigInt!
+
+  "Amount of reward collected by this council member so far."
+  accumulatedReward: BigInt!
+
+  electedInCouncil: ElectedCouncil!
+}
+
+type CandidacyNoteMetadata @entity {
+  "Candidacy header text."
+  header: String
+
+  "Candidate program in form of bullet points."
+  bulletPoints: [String!]!
+
+  "Image uri of candidate's banner."
+  bannerImageUri: String
+
+  "Candidacy description (Markdown-formatted)."
+  description: String
+}
+
+################### Referendum #################################################
+
+type ReferendumStageInactive @variant {
+  # no properties
+
+  # TODO: remove me - variant needs to have at least 1 property now
+  dummy: Int
+}
+
+type ReferendumStageVoting @variant {
+  "Block in which referendum started."
+  started: BigInt!
+
+  "Target number of winners."
+  winningTargetCount: BigInt!
+
+  "Index of current election"
+  electionRound: ElectionRound!
+}
+
+type ReferendumStageRevealing @variant {
+  "Block in which referendum started"
+  started: BigInt!
+
+  "Target number of winners"
+  winningTargetCount: BigInt!
+
+  "Intermediate winning options"
+  intermediateWinners: [ReferendumStageRevealingOptionResult!]!
+
+  "Index of current election"
+  electionRound: ElectionRound!
+}
+
+type ReferendumStageRevealingOptionResult @entity {
+  "Member that received votes."
+  optionId: Membership!
+
+  "Sum of votes' power received."
+  votePower: BigInt!
+
+  # TODO: reference variant (how?)
+  #referendumRevealingStages: [ReferendumStageRevealing!]! @derivedFrom(field: "intermediateWinners")
+
+  "Election round."
+  electionRound: ElectionRound!
+
+  "Event that concluded the referendum."
+  referendumFinishedEvent: ReferendumFinishedEvent!
+}
+
+# TODO: this maybe needs to be @entity - so it's saved in db
+union ReferendumStage = ReferendumStageInactive | ReferendumStageVoting | ReferendumStageRevealing | VariantNone
```

Comment:
This is currently never used

Reply:
Not used items have been commented out until we solve variant references and uncomment all relevant references.